packages/at_secondary_server/config/config.yaml

```diff
@@ -38,6 +38,10 @@ hive:
   expiringRunFrequencyMins: 12
   # List of malformed keys that will be deleted on server startup
   malformedKeys: 'public:publickey'
+  # Delete keys that starts with "public:cached" on server startup
+  # This is set to true by default. To retain the invalid keys on server start-up,
+  # set the flag to false.
+  shouldRemoveInvalidKeys: true
```

Comment:
Incorrect name, does not match code

Reply:
Will update the name and push the changes.